src/js/popup.js

```diff
@@ -333,6 +346,26 @@ $(document).ready(function () {
         });
     });
 
+    $('#rate').click(function () {
+        dmUIClick('rate');
+        chrome.runtime.sendMessage({
+            JDTIfunction: 'setRateStatus'
+        }, function (response) {
+            if (response.status === 'success') {
+                // Flag set
+                window.open('https://chrome.google.com/webstore/detail/jira-template-injector/hmhpegjieopgbdmpocdmfkafjgcdmhha/reviews?hl=en', '_blank'); // Open extension user reviews page
+            } else {
+                if (response.message) {
+                    dmError('export', response.message);
+                    Materialize.toast(response.message, 2000, 'toastNotification');
+                } else {
+                    dmError('export', 'generic');
```

Comment:
Something more descriptive than generic would be better. 'Error saving Rate Now click status'